pkg/common/providers/core/provider.go

```diff
@@ -17,6 +17,7 @@ type CryptoSuiteConfig interface {
 	SecurityProviderPin() string
 	SecurityProviderLabel() string
 	KeyStorePath() string
+	VaultStorePath() string
```

Comment:
This interface change will impact implementors who have a custom configuration provider.